### ISSUE TITLE
constant crashes and errors

### ISSUE THREAD

**Issue by eeshaa1** [Kaldaien/SoulsUnsqueezed#19]
i get an error when i launch DS3T.exe, which has already been reported here. even though the "fix" works I constantly crash. So far I've gotten crashes from receiving items (picking up a sunlight medal, obtaining a boss soul, and receiving the banner from the lady in lothric castle) and lighting bonfires (the first bonfire you light in the tutorial area)

**Comment by Person-Mc-Gee** [comment]
I'm 90% the second part of this is an issue I'm having with the game crashing any time I get an achievement, which all of the items you describe are associated with (except the sunlight medal. are you sure you didn't mean the covenant item you get from the undead settlement?).

I would love to know if you figured anything out about this.

**Comment by eeshaa1** [comment]
yeah i asked in the discord for Special K and you can disable achievement popups. I forgot how though X.X
Join the Discord and ask yourself, then comment how you do it here so anyone else looking for a fix finds it.
